include/aikido/trajectory/util.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef AIKIDO_TRAJECTORY_UTIL_HPP_
+#define AIKIDO_TRAJECTORY_UTIL_HPP_
+
+#include <aikido/trajectory/Interpolated.hpp>
+#include <aikido/trajectory/Spline.hpp>
+
+namespace aikido {
+namespace trajectory {
+
+/// Convert an interpolated trajectory to a piecewise linear spline trajectory
+/// This function requires the \c _inputTrajectory to use a \c
+/// GeodesicInterpolator.
+/// So the conversion exactly preserves the geometric path
+/// \param[in] _inputTrajectory interpolated trajectory
+/// \return a spline trajectory
+std::unique_ptr<aikido::trajectory::Spline> convertToSpline(
+    const aikido::trajectory::Interpolated& _inputTrajectory);
+
+/// Convert a piecewise linear spline trajectory to an interpolated trajectory
+/// using a given interpolator.
+/// \param[in] spline trajectory
+/// \param[in] interpolator used in connecting two ends of a segment
+/// \return an interpolated trajectory
+std::unique_ptr<aikido::trajectory::Interpolated> convertToInterpolated(
+    const aikido::trajectory::Spline& traj,
+    const aikido::statespace::InterpolatorPtr interpolator);
+
+/// Concatenate two spline trajectories
+/// This function converts two spline trajectories into interpolated
+/// trajectories
+/// (geodesic interpolator) for concatenation, then convert the concatenated
```

Comment:
Yes. I originally only said it concatenated two spline trajectories. But later I added more details because I want any future user will be aware of how it works. Because there are a few different strategies in concatenation. This is one of those that works well with the specific need in the servo control.